core/src/main/java/org/bitcoinj/core/PeerGroup.java

```diff
@@ -139,8 +140,8 @@ public class PeerGroup implements TransactionBroadcaster {
      */
     public static final int DEFAULT_CONNECTIONS = 12;
     private volatile int vMaxPeersToDiscoverCount = 100;
-    private static final long DEFAULT_PEER_DISCOVERY_TIMEOUT_MILLIS = 5000;
-    private volatile long vPeerDiscoveryTimeoutMillis = DEFAULT_PEER_DISCOVERY_TIMEOUT_MILLIS;
+    private static final Duration DEFAULT_PEER_DISCOVERY_TIMEOUT_MILLIS = Duration.ofSeconds(5);
```

Comment:
Since this is a private, you could remove the `_MILLIS` from the name.

Reply:
Thanks! This was an oversight – fixed.